hammers/scripts/node_doctor.py

```diff
@@ -0,0 +1,153 @@
+# coding: utf-8
+'''
+.. code-block:: bash
+
+    node-doctor <node_name>
+
+'''
+import argparse
+import re
+import sys
+
+from hammers.scripts.ironic_error_resetter import cureable_nodes
+from hammers.scripts.undead_instances import find_unbound_instances
```

Comment:
If I came back to this codebase, I would not expect scripts to use functions defined within eachother. I think we should leave a comment on the `find_unbound_instances` function saying something like "NOTE: if you modify/delete this function, be sure to check its usage in node_doctor". Or, simply copy the implementation here and remove the coupling.

Reply:
I'll add functionality directly into node doctor